apps/meteor/tests/end-to-end/api/livechat/18-rooms-ee.ts

```diff
@@ -123,5 +131,39 @@ import { IS_EE } from '../../../e2e/config/constants';
 
 			expect(response.body.success).to.be.true;
 		});
+		it('should remove room from hold', async () => {
+			await updatePermission('on-hold-others-livechat-room', ['admin', 'livechat-manager']);
+			const { room } = await startANewLivechatRoomAndTakeIt();
+			await sendAgentMessage(room._id);
+			await setRoomOnHold(room._id);
+			const response = await request
+				.post(methodCall('livechat:resumeOnHold'))
+				.set(credentials)
+				.send({ message: JSON.stringify({ params: [room._id], msg: 'method', method: 'livechat:resumeOnHold', id: 'id' }) })
```

Comment:
I see a bunch of params getting duplicated here. So y not create a helper function to resume chat from on-hold and here just pass the roomId?

Reply:
I've done that already, it's in the data folder, I'm just testing the endpoint directly for completeness reasons.